src/ActionBranchToPaletteCell.test.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2024 Inclusive Design Research Centre, OCAD University
+ * All rights reserved.
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/inclusive-design/adaptive-palette/blob/main/LICENSE
+ */
+
+import { render, screen } from "@testing-library/preact";
+import "@testing-library/jest-dom";
+import { html } from "htm/preact";
+
+import { initAdaptivePaletteGlobals } from "./GlobalData";
+import { ActionBranchToPaletteCell } from "./ActionBranchToPaletteCell";
+
+describe("ActionBranchToPaletteCell render tests", () => {
```

Comment:
Yes, and I decided that the best place to do that is in `Palette.integration.test` since the go-to button affects other areas of the screen and involves other objects.